### DISCUSSION TITLE
save dumped from a SNES ExLorom repro

### DISCUSSION THREAD

**Discussion by Chomemel** [sanni/cartreader#936] [Need help]
I have 2 snes Repro:
- Fire Emblem Thracia (ExLorom, 256 ko SRAM)
- Tales of Phantasia (ExHirom, 64 Ko SRAM)

I was able to dump the ROMS and the SAVES for both of them. 

I have no problem playing the Tales Of Phantasia ROM I dumped with an emulator using the dumped SAVE.

However for the ExLorom Cartride, when I tried to play the ROM with an emulator with the SAVE, the SAVE doesn’t work (the ROM does)

I reflashed the ExLorom with Energy Breaker (4 Mo, so Lorom, 64 ko of SRAM) transforming the PCB to a Lorom one. Playing the game and saving on real hardware worked fine (hourray!). Dumping the ROM and the SAVE worked. But like FE Thracia, no way to use the save on an emulator (with the ROM dumped and also the original ROM). I tried every configurations (A13 and A14 soldered, A13 and A14 unsldered…) before dumping the SAVE. I even erased the SRAM from the repro and write every SAVE I dumped, but no one worked.

Did anyone experience that already? Is it just a problem with my repro?

I kept the SAVES if someone wants to take a look.

**Comment by sanni** [comment]
Could be related to this issue: https://github.com/sanni/cartreader/issues/917

**Comment by Chomemel** [comment]
I erased my post because after some testing, I had different results. I, in fact, didn't know how to use SRM files with snes9X properly.

With Fire Emblem Thracia (ExLorom, 256 K SRAM), it seems that I can dump the save from the repro and play it on the emulator. I can also write a save from emulator and use it on real hardware. For this one, WE is soldered to WE (not Vcc) and so are the jumpers for A13 and A14 (obviously). I'll do some more testing to be 100% sure.

I also tested again with Energy Breaker (Lorom, 64 k), there is only one configuration that works for dumping saves and using it with SNES9X:
- A13 and A14 unsoldered (without a surprise)
- WE conected (not woking if WE is connected to Vcc)

So to end this topic I need to test again starting a game with the emulator and transfer the save to the cartridge (with both Lorom and Exlorom).

**Comment by Chomemel** [comment]
Ok, I finished my tests. Saves from emulator can be loaded et played on my repro (both LoRom and ExLoRom). I just meesed up the first time. Probably because I forgot to turn the OSCR to 5V and also because I learn how to use the saves from snes9X properly the last weeks.

So case closed.
